docs/faq.qmd

```diff
@@ -14,7 +14,17 @@ inference.
 * PyMC is a library for Bayesian modelling, and is the backend used by Bambi. 
 It is a very powerful library, but can be challenging to use for beginners. 
 Bambi provides a simple interface for specifying models, and allows for easy inference via
-MCMC or variational inference using PyMC.
+
+### Why have a Bayesian regression library?
```

Comment:
I am not sure about including this section in the FAQ. Yes, it is related to Bambi, but it is more asking a question of what the right level of abstraction is.